num2words/__init__.py

```diff
@@ -101,8 +102,6 @@ def num2words(number, ordinal=False, lang='en', to='cardinal', **kwargs):
     if lang not in CONVERTER_CLASSES:
         raise NotImplementedError()
     converter = CONVERTER_CLASSES[lang]
-    if isinstance(number, str):
-        number = converter.str_to_number(number)
```

Comment:
Hey @akki2825 you are missing this section. With this conditional, the tests pass.

Reply:
Thank you @lepisma